src/WebJobs.Script/Workers/Rpc/WebHostRpcWorkerChannelManager.cs

```diff
@@ -159,9 +157,9 @@ private bool UsePlaceholderChannel(string workerRuntime)
             // Special case: node and PowerShell apps must be read-only to use the placeholder mode channel
             // Also cannot use placeholder worker that is targeting ~3 but has backwards compatibility with V2 enabled
             // TODO: Remove special casing when resolving https://github.com/Azure/azure-functions-host/issues/4534
-            if (workerRuntime.ToLowerInvariant() is RpcWorkerConstants.NodeLanguageWorkerName
-                or RpcWorkerConstants.PowerShellLanguageWorkerName
-                or RpcWorkerConstants.PythonLanguageWorkerName)
+            if (string.Equals(workerRuntime, RpcWorkerConstants.NodeLanguageWorkerName, StringComparison.OrdinalIgnoreCase)
```

Comment:
Ah yeah forgot about the perf. This might be solved with C# 11 where you can make the invariant via a span and then pattern match against that (no heap allocation, `.ToSpan().ToLowerInvariant()`).

Reply:
I think I'm going to leave this out for now - if we'd like to make changes around this we can create a PR for it with benchmark stats just to be sure!

Reply:
Ah yes, I don't believe we are yet :D